docs/guide/pricing-guide.md

```diff
@@ -0,0 +1,61 @@
+title: Pricing Guide
+description: This guide explains how Sematext Cloud's flexible and transparent pricing works, helping you manage costs effectively based on your needs.
+
+We offer flexible and transparent pricing for all of our four solutions: [Monitoring](https://sematext.com/docs/monitoring/), [Logs](https://sematext.com/docs/logs/), [Experience](https://sematext.com/docs/experience/), and [Synthetics](https://sematext.com/docs/synthetics/). This guide explains how our pricing works, helping you manage costs effectively based on your needs.
+
+## Apps and Plans
+For each of the four solutions, you can create multiple Apps based on your needs. Each App can be tailored to your specific requirements with different plans. This allows you to customize and control costs for each App individually.
+
+- **Logs Apps**: Offer three different plan options with different retention periods and daily log volumes.
+- **Monitoring Apps**: Offer three different plan options with different retention periods and the number of hosts and containers you want to ship metrics from.
+- **Experience Apps**: Offer three different plan options with different retention periods and the number of page views.
+- **Synthetics Apps**: Offer three different plan options with different retention periods, the number of monitors, and run limits for each Browser and HTTP monitor.
+
+This flexibility ensures you only pay for what you use. For example, you might create multiple Apps and choose a minimal retention plan for test hosts and a higher retention plan for production hosts.
+
+Refer to our [cost calculator](https://sematext.com/cost-calculator/) to estimate costs for each solution based on your needs. The calculator sums up the total based on your selections and shows your estimated monthly charge. You can also explore detailed plans on the [pricing page](https://sematext.com/pricing/).
+
+### Can I ship logs from different hosts to the same App without additional charges?
```

Comment:
@fulyauluturk since this is solution-specific, and stuff above and below is for the whole platform, perhaps we should move solution-specific stuff to the bottom.  We may come up with more items, from other solutions, and structure them as FAQ.  For example, we may want to say that we average the number of containers across all monitored hosts. Or we answer the question about per-container hosts. Or we may want to say that plans and retention and other stuff can be changed at any point and any number of times and will take effect immediately. Or we may address the question about spikes in log volume and DLVL. etc.